### PR TITLE
Tic Tac Toe re-style

### PR DESCRIPTION
**Pull request by geoknee**
Complete overhaul of Tic Tac Toe styling. Fixes https://github.com/magmo/apps/issues/96 and resolves https://github.com/magmo/apps/issues/83.

In the wallet landing page, the statestash logo has been replaced by the magmo fireball for now, to remain consistent with the rest of the wallet styling. We should probably have a discussion about overall styling of the wallet, in case we want the statestash styling back in .

### CODE REVIEW

File: packages/tictactoe/src/styles/board.scss [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 #table-container {
     height: 300px;
     width: 300px;
     position: absolute;
     top: 50%;
     left: 50%;
-    transform: translate(-50%, -50%);
+    transform: 
+    translate(-150px, -150px);
+    // translate(0,+$gamebar-height)
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/board.scss]:
```suggestion
```

File: packages/tictactoe/src/styles/board.scss [hunk 2]
```diff
@@ -61,9 +66,9 @@
   
   } */
   
-  .dark {
-    background-color: black;
-  }
+  // .dark {
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/board.scss]: Could we remove the commented out lines?

File: packages/tictactoe/src/styles/footer.scss [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
   font-weight: bold;
   font-size: 16px;
   color: $white;
   width: 188px;
   height: 40px;
   border-radius: 5px;
-  border: solid 1px $rps-lobby-border-off-white;
-  // background-color: $rps-footer-grey;
+  border: solid 1px $ttt-lobby-border-off-white;
+  // background-color: $ttt-footer-grey;
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/footer.scss]:
```suggestion
```

File: packages/tictactoe/src/styles/footer.scss [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
 }
 
 .footer-playagain {
   position: absolute;
   top: 50%;
   left: 50%;
   transform: translate(-50%,-50%);
+  // translate(0,$navbar-height);
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/footer.scss]:
```suggestion
```

File: packages/tictactoe/src/styles/footer.scss [hunk 5]
```diff
@@ -32,27 +36,27 @@
 // .footer-challenge {
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/footer.scss]: Could we remove the commented out code?

File: packages/tictactoe/src/styles/lobby.scss [hunk 6]
```diff
@@ -9,18 +9,20 @@
...
   font-weight: bold;
-  color: $rps-red;
-
+  color: $ttt-white;
   width: 188px;
   height: 40px;
   border-radius: 40px;
-  border: solid 1px $rps-lobby-border-off-white;
-  background-color: $white;
+  // border: solid 1px $ttt-lobby-border-off-white;
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/lobby.scss]:
```suggestion
```

File: packages/tictactoe/src/styles/navbar.scss [hunk 7]
```diff
@@ -45,8 +44,8 @@
   }
   
   .status {
-    background-color: black;
-    font-size: 50px;
+    // background-color: $ttt-background;
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/navbar.scss]: Could we remove commented out code?

File: packages/tictactoe/src/styles/waiting-room.scss [hunk 8]
```diff
@@ -1,18 +1,41 @@
...
     font-size: 18px;
     font-weight: bold;
-    color: $rps-white;
+    color: $ttt-white;
+    width: 188px;
+    height: 40px;
+    border-radius: 40px;
+    border-style: none;
+    background-color: $ttt-orange;
+    // position: absolute;
```
Comment (lalexgap) [inline packages/tictactoe/src/styles/waiting-room.scss]: Could we remove commented out code?

File: packages/tictactoe/src/styles/footer.scss (hunk 5 above)
Comment (geoknee) [reply packages/tictactoe/src/styles/footer.scss]: 👍

**Comment by geoknee** [comment]
Thanks @lalexgap!

**Pull request merged**